openreview/stages/venue_stages.py

```diff
@@ -971,3 +971,131 @@ def __init__(self, committee_id, name='Registration', start_date=None, due_date=
         self.instructions = instructions
         self.title = title
         self.remove_fields = remove_fields
+
+class CustomStage(object):
+    """
+    param reply_to: submission, reviews, metareviews, forum
+    type reply_to: string
+    """
+
+    class Participants(Enum):
+        EVERYONE = 0
+        SENIOR_AREA_CHAIRS = 1
+        SENIOR_AREA_CHAIRS_ASSIGNED = 2
+        AREA_CHAIRS = 3
+        AREA_CHAIRS_ASSIGNED = 4
+        REVIEWERS = 5
+        REVIEWERS_ASSIGNED = 6
+        REVIEWERS_SUBMITTED = 7
+        AUTHORS = 8
+
+    class Source(Enum):
+        ALL_SUBMISSIONS = 0
+        ACCEPTED_SUBMISSIONS = 1
+        PUBLIC_SUBMISSIONS = 2
+
+    class ReplyTo(Enum):
+        SUBMISSION = 0
```

Comment:
what is the difference between submission and forum?

Reply:
Well, `submission` can also be multireply, but it means the replyto is the submission id. 
`Forum` means withForum, so the reply can be made to the submission or to any of it's replies as well.

Reply:
what if we rename them submission -> forum and forum -> withForum?